manual/vscode-extensions/vscode-ember-frost/vscode-ember-frost.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>vscode-ember-frost</id>
+    <title>Ember Colorizer and Theme Extension</title>
+    <version>1.0.0</version>
+    <authors>Ciena Blue Planet</authors>
+    <owners>gep13 chocolatey</owners>
+    <projectUrl>https://marketplace.visualstudio.com/items?itemName=ciena-blueplanet.vsc-ember-frost</projectUrl>
+    <projectSourceUrl>https://github.com/ciena-blueplanet/vscode-ember-colorizer</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/vscode-extensions/vscode-ember-frost</packageSourceUrl>
+    <docsUrl>https://github.com/ciena-blueplanet/vscode-ember-colorizer/blob/master/README.md</docsUrl>
+    <bugTrackerUrl>https://github.com/ciena-blueplanet/vscode-ember-colorizer/issues</bugTrackerUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Visual Studio Code Ember Colorizer and Theme Extension</summary>
+    <description>VsCode extension that colorizes/tokeninzes Ember .hbs, Controllers, and Routes files.</description>
+    <tags>Microsoft VisualStudioCode vscode Ember Extension foss</tags>
```

Comment:
Tags in lowercase as per [§1.2.2](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#122-obligatory-tags)